### DISCUSSION TITLE
Add support for Solar/Powerwall

### DISCUSSION THREAD

**Discussion by knewmania** [teslamate-org/teslamate#615] [Ideas]
**Is your feature request related to a problem? Please describe.**

This feature request is not related to a problem, just a request for additional functionality.

**Describe the solution you'd like**

Would like the TeslaMate solution to support Tesla Solar and Powerwall integration. Provide the ability to query the Tesla API related to Solar and Powerwall features. This will allow for graphing of Solar and Powerwall related data graphing.

**Describe alternatives you've considered**

N/A

**Additional context**

N/A

**Comment by knewmania** [comment]
Adding some resources:

Tesla Powerwall Local Gateway API Documentation:
https://github.com/vloschiavo/powerwall2

Tesla API Commands:
https://www.teslaapi.io/powerwalls/commands

**Comment by SemoTech** [comment]
@knewmania I would love to see this included as well.  TeslaMate is the best!

**Comment by gt2690b** [comment]
any traction on this?
